Flabbergasted-Flamingos/bangazon-api
# added departments in api

# Description
Add description about what this pull request covers.

## Number of Fixes
Type of number of fixes included in this PR.

## Related Ticket(s)
List related ticket(s) that apply to the work done in this pull request

## Problem to Solve
Describe the problem(s) this pull request solves.

## Proposed Changes
Describe the proposed changes.

## Expected Behavior
Describe the expected behavior.

## Steps to Test Solution

```
git fetch --all
git checkout cd-department
rm -r api/migrations
rm -r db.sqlite3
python manage.py makemigrations api
python manage.py migrate
python manage.py runserver
```
then check to see if department works

## Testing

[ ] There are new unit tests in this PR, and I verify that there is full coverage of all new code.
[ ] I certify that all existing tests pass

## Documentation
[ ] There is new documentation in this pull request that must be reviewed..
[ ] I added documentation for any new classes/methods

## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any
db migrations, etc.
